<launch>
  <node name="laser_scanner_base_frame_mgr" type="static_transform_publisher" pkg="tf" args="0 0 .5 0 0 0 1 /base_link /laser_link 50" />
  <node pkg="base_scan" type="base_scan_node" name="base_scan_node"
        output="screen">
  <rosparam command="load" file="$(find base_scan)/config/config.yaml" />
  </node>
</launch>